app/sensors/TextField.js

```diff
@@ -148,16 +148,15 @@ export default class TextField extends Component {
 			"rbc-title-active": this.props.title,
 			"rbc-title-inactive": !this.props.title,
 			"rbc-placeholder-active": this.props.placeholder,
-			"rbc-placeholder-inactive": !this.props.placeholder
+			"rbc-placeholder-inactive": !this.props.placeholder,
+			[this.props.className]: this.props.className
```

Comment:
You don't need to write it like `[foo]: foo`, you can simply write it as:

```
const cx = classNames({...}, foo);
```

Reply:
Cool. Didn't know about that syntax. Updated.